### ISSUE TITLE
If this message I use a custom component. can I hide the toast when clicking a button in my custom component?

### ISSUE THREAD

**Issue by sxs1995** [arnnis/react-native-fast-toast#18]
Use the hook, If this message I use a custom component. can I hide the toast when clicking a button in my custom component? I cannot get this id.

Any help will be greatly appreciated.

![image](https://user-images.githubusercontent.com/22333956/101882503-f5eb3180-3bd0-11eb-8299-a84cc278dec3.png)

**Comment by arnnis** [comment]
Thanks for opening issue.

It's not possible currently, you need the toast id.

I've some idea to address this. will let you know when ready.

**Comment by arnnis** [comment]
I just published a new update [v2.2.0](https://github.com/arnnis/react-native-fast-toast/releases/tag/v2.2.0)

Now you can define a custom id for your toasts:

```
let id = Math.random().toString()

toast.show(
   <Text onPress={() => toast.hide(id)}>Press to hide</Text>, 
   { id }
)
```

Hope this helps.

**Comment by mthines** [comment]
Here's a proposed solution :) 
#48
